From: SeongJae Park <sj@kernel.org>
Date: Thu, 9 Jan 2025 09:25:01 -0800
Subject: [PATCH] Revert "Docs/ABI/damon: document DAMOS filter pass sysfs
 file"

This reverts commit 5aac7e90c5aa29edba6b64ba5f9733aa4aa69c1e.
---
 Documentation/ABI/testing/sysfs-kernel-mm-damon | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/Documentation/ABI/testing/sysfs-kernel-mm-damon b/Documentation/ABI/testing/sysfs-kernel-mm-damon
index 688bd27d8602..8c0acb31638b 100644
--- a/Documentation/ABI/testing/sysfs-kernel-mm-damon
+++ b/Documentation/ABI/testing/sysfs-kernel-mm-damon
@@ -355,16 +355,10 @@ Description:	If 'target' is written to the 'type' file, writing to or
 What:		/sys/kernel/mm/damon/admin/kdamonds/<K>/contexts/<C>/schemes/<S>/filters/<F>/matching
 Date:		Dec 2022
 Contact:	SeongJae Park <sj@kernel.org>
-Description:	Writing 'Y' or 'N' to this file sets whether the filter is for
-		the memory of the 'type', or all except the 'type'.
-
-What:		/sys/kernel/mm/damon/admin/kdamonds/<K>/contexts/<C>/schemes/<S>/filters/<F>/pass
-Date:		Dec 2024
-Contact:	SeongJae Park <sj@kernel.org>
-Description:	Writing 'Y' or 'N' to this file sets whether to allow (pass
-		through the filter) or deny (be blocked by the filter) applying
-		the scheme's action to the memory that satisfies the 'type' and
-		the 'matching' of the directory.
+Description:	Writing 'Y' or 'N' to this file sets whether to filter out
+		pages that do or do not match to the 'type' and 'memcg_path',
+		respectively.  Filter out means the action of the scheme will
+		not be applied to.
 
 What:		/sys/kernel/mm/damon/admin/kdamonds/<K>/contexts/<C>/schemes/<S>/stats/nr_tried
 Date:		Mar 2022
-- 
2.39.5

